packages/fx-core/resource/package.nls.json

```diff
@@ -632,6 +632,7 @@
   "error.common.MissingRequiredFileError": "Missing %srequired file `%s`",
   "error.common.HttpClientError": "A http client error happened while performing the %s task. The error response is: %s",
   "error.common.HttpServerError": "A http server error happened while performing the %s task. Please try again later. The error response is: %s",
+  "error.common.ConcurrentError": "Previous task is still running. Please wait until your previous task to finish and try again.",
```

Comment:
Please -> remove
to finish -> is finished

Reply:
Will fix in another PR.